synfig-studio/src/gui/docks/dock_timetrack2.cpp

```diff
@@ -233,6 +233,29 @@ void Dock_Timetrack2::setup_tool_palette()
 		action_button_map[tool_button->get_name()] = tool_button;
 		tool_item_group->add(*tool_button);
 	}
+
+	std::vector<std::string> stock_id{ "synfig-interpolation_type_clamped", "synfig-interpolation_type_tcb",
+								  "synfig-interpolation_type_const", "synfig-interpolation_type_ease" ,
+								  "synfig-interpolation_type_linear"};
+	std::vector<std::string> name{ "Clamped", "TCB" , "Constant" , "Ease In/Out" , "Linear"};

```

Comment:
To mark string as translatable, but without translating in this code point, use `N_()`, like `N_("Clamped")`.
It does nothing at runtime, but it makes the translatable string parser (i.e. gettext) know that string must be included in .pot/.po files.

Reply:
alright makes sense thanks!